CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased-`x.y.z`] - 2020-xx-xx
 
 ### Features:
+- Updated the version of the local API service used by the UnrealGDK.
```

Comment:
Shouldn't this be a bugfix which contains something like "Fixed a bug which caused the local API service to memory leak"

Reply:
Good point, I've added this in all three branches.